Main.c

```diff
@@ -0,0 +1,17 @@
+#include "CheckRange.h"
+#include <assert.h>
+
+int main()
+{
+int *range;
+
+int array[] = {3, 3, 5, 4, 10, 11, 12};

```

Comment:
test case need to consider variations in input , for ex. {3} , {4.4}, { 3,5,6,7,8}